liquid-staking/src/common/storage.rs

```diff
@@ -0,0 +1,71 @@
+use multiversx_sc::imports::*;
+pub type Epoch = u64;
+
+#[multiversx_sc::module]
+pub trait CommonStorageModule {
+    #[view(getDelegationAddress)]
+    #[storage_mapper("delegationAddress")]
+    fn delegation_addresses(
+        &self,
+        contract_name: &ManagedBuffer,
+    ) -> SingleValueMapper<ManagedAddress>;
+
+    #[view(getDelegatedValue)]
+    #[storage_mapper("delegatedValue")]
+    fn delegated_value(&self, validator: &ManagedAddress) -> SingleValueMapper<BigUint<Self::Api>>;
+
+    #[view(unDelegateEpoch)]
+    #[storage_mapper("unDelegateEpoch")]
+    fn undelegate_epoch(&self, address: &ManagedAddress) -> SingleValueMapper<Epoch>;
+
+    #[view(getTotalEgldSupply)]
+    #[storage_mapper("totalEgldSupply")]
+    fn egld_token_supply(&self) -> SingleValueMapper<BigUint>;
+
+    #[view(getHeaderVerifierAddress)]
+    #[storage_mapper("headerVerifierAddress")]
+    fn header_verifier_address(&self) -> SingleValueMapper<ManagedAddress>;
+
+    #[view(getRegisteredBlsKeys)]
+    #[storage_mapper("registeredBlsKeys")]
+    fn registered_bls_keys(&self) -> UnorderedSetMapper<ManagedBuffer>;
+
+    #[view(validatorBlsKeyMap)]
+    #[storage_mapper("validatorBlsKeyMap")]
+    fn validator_bls_key_address_map(
+        &self,
+        bls_key: &ManagedBuffer,
+    ) -> SingleValueMapper<ManagedAddress>;
```

Comment:
Have a custom type for bls keys, i.e. `pub type BlsKey<M> = ManagedBuffer<M>;`

Makes the code a lot more readable, and it's very easy to change the type if you ever need to.